PORTAL/jobs/__init__.py

```diff
@@ -598,7 +615,7 @@ def _build_send_script(self, queue_log, pushfsfields=None, pullfsfields=None, *,
             pvalue = _utils.quote_shell_str(pvalue)
             bvalue = bfiles.look_up(area, name)
             _utils.check_shell_str(bvalue)
-            pushfiles.append((bvalue, pvalue))
+            pushfiles.append((pvalue, bvalue))
```

Comment:
Nice catch!

Reply:
Yeah -- it's only evident if portal_user != bench_user or portal OS != bench OS.  ;)